corehq/sql_accessors/sql_templates/soft_undelete_forms.sql

```diff
@@ -0,0 +1,19 @@
+DROP FUNCTION IF EXISTS soft_undelete_forms(TEXT, TEXT[], TEXT);
+
+CREATE FUNCTION soft_undelete_forms(
+    p_domain TEXT,
+    form_ids TEXT[],
+    p_problem TEXT,
```

Comment:
didn't think about adding that to an XFormOperationSQL, but now thinking of it i like the idea